setuptools/_distutils/dist.py

```diff
@@ -1221,6 +1223,11 @@ def get_license(self):
 
     get_licence = get_license
 
+    def get_license_expression(self):
+        return self.license_expression
+
+    get_license_expression = get_license_expression
+
```

Comment:
Would it be possible to implement this feature without changing the `_distutils` folder?

In general, we don't modify the contents to that folder directly. Instead, the code is maintained in the `pypa/distutils` repository and merged back in setuptools periodically,  ([see also our development guidelines](https://setuptools.pypa.io/en/latest/development/developer-guide.html#code-conventions-and-other-practices)).

Reply:
> Would it be possible to implement this feature without changing the `_distutils` folder?

Yes. Removed those changes as they weren't necessary to begin with. I added them initially but the better solution was to add `license_expression` to `_DISTUTILS_UNSUPPORTED_METADATA` in `setuptools/dist.py`. This is also fully compatible with `SETUPTOOLS_USE_DISTUTILS=stdlib`. https://github.com/pypa/setuptools/pull/4706/commits/390b95f2b904bd75df48fa64e6370bedbfb63457